src/hypothesis/_settings.py

```diff
@@ -421,13 +421,10 @@ def _validate_timeout(n):
 limit - Hypothesis won't e.g. interrupt execution of the called
 function to stop it. If this value is <= 0 then no timeout will be
 applied.
-
-Note: This setting is deprecated. In future Hypothesis will be removing the
-timeout feature.
 """,
     deprecation_message="""
-The timeout feature will go away in a future version of Hypothesis. To get
-the future behaviour set timeout=hypothesis.unlimited instead (which will
+This setting is deprecated and will be removed in a future version. To get
```

Comment:
This is now kinda an unhelpful message because it gets emitted when people use the timeout setting, but that is usually going to be one of several settings being used in creation and it just says "this setting" without disambiguating.